src/main/java/seedu/address/commons/core/Alias.java

```diff
@@ -41,6 +44,9 @@ public Alias(String aliasName, String command) {
 
     /**
      * Validates alias name.
+     *
+     * @param aliasName name of the alias.
+     * @return whether the name matches the regex pattern.
```

Comment:
Perhaps the first letter of the parameter description and return description should be capitalized?

That is,
`@param aliasName Name of the alias.`
`@return Whether the name matches the regex pattern.`

Saw this in most other files.